npm/qsharp/src/language-service/language-service.ts

```diff
@@ -26,8 +26,7 @@ import {
 } from "../workers/common.js";
 type QscWasm = typeof import("../../lib/web/qsc_wasm.js");
 
-// Only one event type for now
-export type LanguageServiceEvent = {
+export type LanguageServiceDiagnosticEvent = {
```

Comment:
Wow I had totally forgotten that diagnostics were `Event`s at this layer. I'm so happy/relieved to rediscover this. Since we're not plumbing callback arguments through seven layers of JavaScript, it's not going to be a QCOM breaking change :)